frontend/components/AddBookForm/AddBookForm.js

```diff
@@ -29,112 +29,109 @@ const AddBookForm = ({ onClick }) => {
 
   const findAuthor = (fName, lName) => {
     return authors.find(
-      (author) => author.firstName === fName && author.lastName === lName
+      author => author.firstName === fName && author.lastName === lName
     );
   };
 
-  const submitForm = async (data) => {
+  const submitForm = async data => {
     const title = data.Title;
     const description = data.Description;
-    const fName = data["First Name"];
-    const lName = data["Last Name"];
+    const fName = data['First Name'];
+    const lName = data['Last Name'];
 
-    let targetAuthor = {};
-
-    targetAuthor = findAuthor(fName, lName);
-    if (typeof targetAuthor === "undefined") {
-      targetAuthor = await addAuthor(fName, lName).catch((err) => {
+    let targetAuthor = findAuthor(fName, lName);
+    if (typeof targetAuthor === 'undefined') {
```

Comment:
I see what you are trying to do here, by checking if it's undefined, but you don't need to use the `typeof` keyword.. Perhaps you can do this without it?

There is a shorter way to write this